src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java

```diff
@@ -42,6 +42,21 @@ void testMappingUnsupportedTypesReturnsUnsupportedType(final int jdbcType) {
         assertThat(this.reader.mapJdbcType(jdbcTypeDescription).getExaDataType(), equalTo(ExaDataType.UNSUPPORTED));
     }
 
+    @ValueSource(ints = { Types.NUMERIC })
```

Comment:
Why a parametrized test?

Reply:
You're right. Forgot to refactor it after removing types